application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -82,12 +85,13 @@
             "852509694341283871", DOODLECREW_NAME, "878067389634314250", WORDSNACK_NAME,
             "879863976006127627", LETTERTILE_NAME, "879863686565621790");
 
-    private static final List<OptionData> inviteOptions = List.of(
-            new OptionData(OptionType.STRING, MAX_USES_OPTION,
-                    "The amount of times the invite can be used, default is infinity", false),
+    private static final List<OptionData> inviteOptions = List.of(new OptionData(OptionType.INTEGER,
+            MAX_USES_OPTION,
+            "How many times this invite can be used, 0 infinite (default) - 100 being the highest.",
+            false).setRequiredRange(0, MAX_USES_LIMIT),
             new OptionData(OptionType.INTEGER, MAX_AGE_OPTION,
-                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false));
-
+                    "How long, in days this activity can be used before it expires, 0 (No expiry), Maximum is 7 days. ",
+                    false).setRequiredRange(0, MAX_AGE_DAYS_LIMIT));
```

Comment:
You wrote `100` and `7` in the text directly. But what if someone changes the constants (lets say 50 and 6), I bet they would forget to also adjust the text...

It would be better to let the text use the variables instead `"... - " + MAX_USES_LIMIT + " being the highest."` etc. If it starts to become unreadable, use `String.format("... %d ...", foo)` instead.